server-rs/src/leagues.rs

```diff
@@ -17,8 +17,8 @@ impl LeaguesQuery {
             League,
             r#"
                 SELECT id, name, matchmaking_type as "matchmaking_type: _", description,
-                    signups_after, start_at, end_at, badge_path, image_path, rules_and_info,
-                    link
+                    signups_after, start_at, end_at, badge_path as badge_path_internal,
```

Comment:
My preference would be to leave these as `_path` (still `graphql(skip)`) and add `_url` computed fields instead. We don't really need to maintain the same structure as the older APIs, as common components should generally have GQL wrapper versions that expose fragments to request the data they need (and thus would know how to map their own fragment into whatever underlying UI components)